src/elements/core/base-elements/button-base-element.ts

```diff
@@ -96,11 +96,11 @@ export abstract class SbbButtonBaseElement extends SbbActionBaseElement {
   private _dispatchClickEventOnSpaceKeyup = (event: KeyboardEvent): void => {
     if (event.key === ' ') {
       this._removeActiveMarker(event);
-      this._dispatchClickEvent(event);
+      this.dispatchClickEvent(event);
     }
   };
 
-  private _dispatchClickEvent = (event: KeyboardEvent): void => {
+  protected dispatchClickEvent = (event: KeyboardEvent): void => {
```

Comment:
This change is no longer needed. If this functionality is needed externally, we can extract it as a function or as a static method.